rules/deb_packages.bzl

```diff
@@ -54,14 +51,17 @@ _deb_packages = repository_rule(
         "arch": attr.string(
             doc = "the target package architecture, required - e.g. arm64 or amd64",
         ),
+        "timestamp": attr.string(
+            doc = "the timestamp value of the archive lookup (yyyyMMddTHHmmssZ), required - e.g. 20091004T111800Z",
```

Comment:
doc says attribute is required. Is this necessary?

For debian/ubuntu upstream packages, I can see the need given the reaping policies in case a package is superseded. 

For smaller third party repositories, this is not always the case. E.g. https://haproxy.debian.net/pool/main/h/haproxy/

Reply:
Indeed it's not necessarily needed. If i remember correctly by default the attributes are not mandatory in bazel. I will test and update the doc.
But i will have to add a test to the update procedure to check if a timestamp is present or not as right now i will add the timestamp at each update. To be consistent to the fact that if the attribute is not required, it should not be added automatically.

Reply:
I made the following change in order to have a safe timestamp attribute not mandatory :

- updated the documentation
- update timestamp attribute value in the update_deb_packages go code only if a value was defined before
- Check in the bazel rule if a timestamp attribute is not defined and the templating on timestamp used inside an url. If it's the case -> fail